src/app/assumptions-and-limitation/assumptions-and-limitation.component.html

```diff
@@ -95,23 +95,25 @@ <h4 class="tce-text-lg">Power consumption</h4>
   <h4 class="tce-text-lg">Carbon Intensity</h4>
   <p class="tce-text-sm">
     We make use of the latest available Carbon Intensity figures from the
-    <a class="tce-underline" href="https://ember-climate.org/data/data-tools/data-explorer/">Ember Data Explorer</a> for
-    the regions we make available. At present these are:
+    <a class="tce-underline" href="https://ember-climate.org/data/data-tools/data-explorer/">Ember Data Explorer</a> via
+    the CO2.js library. We limit the range of regions that can be selected, which are currently:
```

Comment:
Is it worth adding a link to the CO2.js library here? Also, not something we can do anything about but it's taking a long time to connect to the Ember website for me at the moment (around 30 seconds). My latest attempt to connect (via Google) actually timed out altogether. Might just be a temporary thing but maybe worth keeping an eye on.